docs/interface.rst

```diff
@@ -385,9 +385,29 @@ The follow event kinds are understood (see ``status.py`` for more details on the
 
 - ``"download-finished"``: same as upload version.
 
+- ``"invite-created"``: A new invite is created. All invite events also have a ``folder`` indicating which folder they pertain to, a ``id``, ``participant-name`` indicating the invitee and ``mode`` (``"read-only"`` or ``"read-write"``).
+
+- ``"invite-welcomed"``: Connection to the mailbox has succeeded. As well as the keys from ``"invite-created"`` the welcome data is contained in ``welcome`` (you should show users the ``"motd"`` if there is one).
+
+- ``"invite-code-created"``: As well as the keys from ``"invite-created"``, a ``code`` key will be included indicating the wormhole code.
+
+- ``"invite-versions"``: As well as the keys from ``"invite-created"``, a ``versions`` key is included indicating the peer's version information.
```

Comment:
This event-kind follows a different naming-convention than the others (which use `object-verb`, in past-tense, throughout). Consider updating/changing this to match the existing convention (though, in this case, I'm not sure what would make the most sense -- "invite-version-received"? "invite-version-negotiated"? "invite-version-accepted"?).

Reply:
This just means "we got the `versions` message from the mailbox" -- does that help with naming?

Reply:
Perhaps "invite-version-received" then?